Swinject/Container.swift

```diff
@@ -100,7 +100,7 @@ public final class Container {
     ///
     /// - parameter loader: the loader to load properties into the container
     ///
-    public func applyPropertyLoader<T: PropertyLoaderType>(loader: T) throws {
+    public func applyPropertyLoader(loader: PropertyLoaderType) throws {
```

Comment:
In the `Assembler` you can pass an array of property loaders that will be applied, in order, to the container before the assemblies are loaded. Since the signature of the ```init``` function for the ```Assembler``` cannot constrain the array of property loaders generically it has be be passed as so:
```
init(...., propertyLoaders: [PropertyLoaderType], ...)
```
This created a problem for ```Container:applyPropertyLoader``` because the value passed for the ```loader``` param can only be constrained as a ```PropertyLoaderType``` not the type itself. So I had to make this change which doesn't break the API.